tests/docker/run_tests.sh

```diff
@@ -30,6 +30,6 @@ if [ "$REBUILD" == "t" ]; then
 fi
 
 if ${SCRIPT_DIR}/ducker-ak ssh | grep -q '(none)'; then
-    ${SCRIPT_DIR}/ducker-ak up -n "${KAFKA_NUM_CONTAINERS}" || die "ducker-ak up failed"
+    ${SCRIPT_DIR}/ducker-ak up -j 'openjdk:11' -n "${KAFKA_NUM_CONTAINERS}" || die "ducker-ak up failed"
```

Comment:
Do we intend to change this temporarily or permanently?

Reply:
I think this should be reverted prior merge. 
Changed only to test PR on external server(my local machine too slow to run all changed tests)